src/Components/Publishing/EditorialFeature/Components/Eoy2018Artists/ArticleHeader.tsx

```diff
@@ -32,9 +33,12 @@ export class Eoy2018ArticleHeader extends React.Component<{
   }
 
   imagesGrid = gridSize => {
-    const { isMobile, isTablet } = this.props
+    const { isMobile, isTablet, isTest } = this.props
     const imageUrls = this.getImageUrls(gridSize)
 
+    if (isTest) {
```

Comment:
Not sure what `isTest` mean in this case, maybe leave a comment

Reply:
Left a comment on the PR, will add one to the code -- because the images are randomized it is impossible to get a dependable snapshot, but we definitely want snapshots to watch for degredation over time... this is used to turn off the grid during testing. Is a little weird but I think useful for this case.